config.py

```diff
@@ -93,7 +93,7 @@
     "https://www.data.gouv.fr/fr/datasets/r/2f4f901d-e3ce-4760-b122-56a311340fc4"
 )
 URL_COLTER_EPCI = (
-    "https://www.collectivites-locales.gouv.fr/files/2022/epcisanscom2022.xlsx"
+    "https://www.collectivites-locales.gouv.fr/files/Accueil/DESL/2024/epcicom2024.xlsx"
```

Comment:
Worth to try to variablize the year. And if it does not work on new year, you may
 : 
- fallback on previous year url
- raise an exception and be notify to update it

Reply:
You chose violence today 😅